README.md

```diff
@@ -1125,18 +1125,24 @@ Options for the [OpenID Connect](http://developer.okta.com/docs/api/resources/oi
 
 ### Smart Card IdP
 
-**:information_source: EA feature:** The Smart Card IdP feature is currently an [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea). This feature is only available for okta-hosted widget implementations.
+**:information_source: EA feature:** The Smart Card IdP feature is currently an [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea).
 
 Settings for authentication with a Smart Card `X509` type IdP.
 
+- `certAuthUrl` *(required)* - [MTLS SSO Endpoint](https://developer.okta.com/docs/reference/api/idps/#mtls-single-sign-on-sso-endpoint-object) that requests the browser for a certificate and performs authentication.
+
 - `text` *(optional)* - Label for the Smart Card IdP button. By default, this value will be "Sign in with PIV / CAC card".
 
 - `className` *(optional)* - Class that can be added to the Smart Card IdP button.
 
+- `isCustomDomain` *(optional)* - Boolean that indicates if the request is coming from a custom domain.
```

Comment:
Do we need to mention what it defaults to?

Reply:
This just passes on whatever value the widget gets. There is no default as such.. Is this something we need to change?

Reply:
It does JSON.stringify before post though https://github.com/okta/okta-signin-widget/blob/master/src/VerifyPIVController.js#L74

Reply:
Asking for @mauriciocastillosilva-okta 's opinion as well. I don't think we need to change this to have a default, because if it is not provided / undefined, it is not sent in the request

Reply:
I guess we can add something to explain it right? If I am understanding this right, this will default to false if not provided. https://github.com/okta/okta-signin-widget/blob/master/src/VerifyPIVController.js#L37

So we can add that "if not provided, default value is false"

Reply:
if not provided, it will be undefined and removed from request data. There is a test for it - https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/VerifyPIV_spec.js#L133

Reply:
Can we say something that indicates what the behaviour effectively defaults to? Will the following work?

> If omitted, defaults to `false`.

Reply:
If omitted, it is removed from the request data by JSON.stringify. 🙂 see comment here - https://github.com/okta/okta-signin-widget/pull/1306#discussion_r455447406

Reply:
Yes, I saw that. I just feel that the doc would look more complete if we talk about what the user should expect in _every_ case.

Do we have existing examples in the SIW doc that talk about default behaviours when a parameter is omitted (and removed from request data)?

Reply:
Looked in the docs and found some examples of omitted. Nothing specific to our case though. Updated the doc with the behavior when omitted. Please let me know what you think.